.travis.yml

```diff
@@ -5,3 +5,4 @@ before_script:
   - npm install -g gulp
 script:
   - npm run build
```

Comment:
Sure. Will update the gulp file. This PR is for testing only. Right now it shows all the image comparison failing. Trying to figure it out.. Any suggestions?